README.md

```diff
@@ -24,15 +23,25 @@ The `pulumilocal` command has the same usage as the `pulumi` command. For detail
 please refer to the man pages of `pulumi -h`.
 
 For example:
-### Create a new Pulumi Project with Stack name lsdev
+
+### Add environment variables to store state on local backend (optional)
 ```shell
-mkdir myproj
 export PULUMI_CONFIG_PASSPHRASE=lsdevtest
 export PULUMI_BACKEND_URL=file://`pwd`/myproj
+```
+_Note: For further options please consult the official documentation on available [environment variables][env_vars] and [local backend][local_backend]._
+
+[env_vars]: https://www.pulumi.com/docs/cli/environment-variables/
+[local_backend]: https://www.pulumi.com/docs/concepts/state/#local-filesystem
+
+### Create a new Pulumi Project with stack name lsdev
+```shell
+mkdir myproj
 pulumilocal new typescript -y -s lsdev --cwd myproj
 ```
+_Note: `--cwd` switch is unnecessary if commands are being run in project directory._
 
-### Select and Create the lsdev Pulumi Stack
+### Select and Create the lsdev Pulumi stack
```

Comment:
small nit: if we lowercase `stack`, then we could also change the casing of `create`:
```suggestion
### Select and create the lsdev Pulumi stack
```